Userland/Libraries/LibWeb/DOMTreeModel.h

```diff
@@ -33,6 +33,8 @@ class DOMTreeModel final : public GUI::Model {
     virtual GUI::ModelIndex index(int row, int column, const GUI::ModelIndex& parent = GUI::ModelIndex()) const override;
     virtual GUI::ModelIndex parent_index(const GUI::ModelIndex&) const override;
 
+    GUI::ModelIndex index_for_node(u64 node_id);
```

Comment:
Can this be `const` qualified?

Reply:
Yup, it can. My brain still doesn't think in `const`-ness yet.